webpack.config.js

```diff
@@ -44,18 +30,18 @@ const webConfig = dev.getDefaultWebpackConfig({
         '../build/default/validation': 'commonjs ../build/default/validation',
         '../build/Release/bufferutil': 'commonjs ../build/Release/bufferutil',
         '../build/default/bufferutil': 'commonjs ../build/default/bufferutil',
+
+        // required for cloud shell feature
+        bufferutil: 'commonjs bufferutil',
```

Comment:
This is the scary stuff 😬

Reply:
Honestly I don't know what this does

Reply:
lmao I'm pretty sure it's polyfilling. Probably has to do with cloud shell stuff running on its own node process.